src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  $root:&;
+
+  padding: $paddings;
+  display: flex;
+  flex-wrap: wrap;
+  width: calendarWidth($sideSize, $space, $paddings);
+  gap: 1px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100px;
+    height: 100px;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition: background-color 0.5s, transform 0.5s;
+    }
+
+    @for $index from 1 to 32 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+      }
+    }
```

Comment:
Shouldn't it be 1 to 31?

Reply:
if use TO -> 32
if use THROW -> 31

Reply:
Oh, nice to know. Thank you for explanation!